docs/references/strategies/languages/gradle/gradle.md

```diff
@@ -338,7 +338,7 @@ targets:
 ```
 
 
-1) Running `fossa analyze --output -c .fossa.yml`, will only analyze `utilities` submodule.
+1) Running `fossa analyze --output -c .fossa.yml`, will only analyze `list` submodule.
 
```

Comment:
Not part of the ticket, but I think this is wrong based on the config above so I'm fixing it